clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3301,6 +3301,25 @@ def IterEndOp : CIR_Op<"iterator_end"> {
 // Floating Point Ops
 //===----------------------------------------------------------------------===//
 
+class UnaryFPToIntBuiltinOp<string mnemonic> : CIR_Op<mnemonic, [Pure]> {
+  let arguments = (ins CIR_AnyFloat:$src);
+  let results = (outs CIR_IntType:$result);
+
+  let summary = [{
+    libc builtin function that takes a floating-point value and produces an
```

Comment:
Because we already have operations that map libc 1-1 (e.g. `cir.libc.memchr`), maybe just improve the comment a bit to perhaps mention equivalence but not direct mapping.

Reply:
I think the summary now is too general. Maybe it's better to write summary for each specific operation?

Reply:
Generic is fine, a quick change to just made the distinction I mention should be good enough for now! I suggest:
```
Builtin function that takes a floating-point value and produces an integral value.
```
I don't know if this works, but if we could use `$_op->getName` as part of the desc, that'd be cool too.

Reply:
Updated. `$_op->getName` seems not working.